packages/website/components/account/filesManager/uploadsTable.js

```diff
@@ -264,114 +389,35 @@ const UploadsTable = ({ content, hidden, onFileUpload, onUpdatingChange, showChe
-              onClick={onFileUpload}
-              variant={content?.table.cta.theme}
-              tracking={{
-                ui: countly.ui[content?.table.cta.ui],
-                action: content?.table.cta.action,
-                data: { isFirstFile: true },
-              }}
-              disabled={info?.tags?.['HasAccountRestriction']}
-              tooltip={info?.tags?.['HasAccountRestriction'] ? content?.table.cta.accountRestrictedText : ''}
-            >
-              {content?.table.cta.text}
-            </Button>
-          </span>
-        ) : (
-          uploads.map(item => (
-            <FileRowItem
-              key={item.cid}
-              onSelect={() => onUploadSelect(item)}
-              date={item.created}
-              name={item.name}
-              cid={item.cid}
-              status={
-                Object.values(PinStatus).find(status => item.pins.some(pin => status === pin.status)) ||
-                PinStatus.QUEUING
-              }
-              storageProviders={
-                Array.isArray(item.deals)
-                  ? item.deals
-                      .filter(deal => !!deal.storageProvider)
-                      .map((deal, indx, deals) => (
-                        <span key={deal.dealId}>
-                          <a
-                            className="underline"
-                            href={`https://filfox.info/en/deal/${deal.dealId}`}
-                            target="_blank"
-                            rel="noreferrer"
-                          >
-                            {`${deal.storageProvider}`}
-                          </a>
-                          {indx !== deals.length - 1 && ', '}
-                        </span>
-                      ))
-                  : null
-              }
-              size={'dagSize' in item ? filesize(item.dagSize) : '-'}
-              linkPrefix={linkPrefix}
-              highlight={{ target: 'name', text: keyword?.toString() || '' }}
-              numberOfPins={item.pins.length}
-              isSelected={!!selectedUploads.find(fileSelected => fileSelected === item)}
-              onDelete={() => onDeleteSingle(item.cid)}
-              isEditingName={item.cid === nameEditingId}
-              onEditToggle={onEditToggle(item.cid)}
-              tabType="uploaded"
-            />
-          ))
-        )}
-      </div>
-      {!!uploads.length && (
-        <div className="files-manager-footer">
+
+      <Table
+        columns={columns}
+        rows={uploads.map(file => fileToTableRow(file))}
+        totalRowCount={totalUploads}
+        page={page}
+        totalPages={pages}
+        rowsPerPage={size}
+        rowsPerPageOptions={ROWS_PER_PAGE_OPTIONS}
+        isEmpty={totalUploads === 0}
+        withRowSelection={true}
+        isLoading={isFetchingUploads || !fetchDate}
+        onPageSelect={onPageSelect}
+        onSetItemsPerPage={setSizeHandler}
+        emptyState={<span>EMPTY</span>}
```

Comment:
We need to align with the production empty state here.

Reply:
Great spot! Addressed 👍

Reply:
Good spot! Addressed, thanks